docs/developer/app-store/overview.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+---
+
+:::info
+
+This feature requires Saleor version **3.12**.
+
+:::
+
+Saleor App Store is a [Saleor Cloud](cloud.mdx) feature that provides an instantly installable collection of [apps](developer/extending/apps/key-concepts.mdx). Each app brings new functionality to Saleor, often by integrating with third-party services.
+
+You can find Saleor App Store under the _Apps_ tab in the Dashboard menu. [Here](https://demo.saleor.io/dashboard/apps/) is Saleor App Store for the demo instance.
+
+## Usage
+
+Saleor App Store consists of two sections:
+
+- Installed Apps - a list of apps that are currently installed on your instance
+- All Apps - a list of all apps available in the App Store
```

Comment:
That is not true. Technically *all* Saleors (not only cloud) have "installed apps" and only Cloud has access to "All apps" section below, which is actual appstore

So we need to either remove that Appstore is "cloud only" or remove mention of installed apps

I'd follow the latter approach and rephrase it (something like "appstore is a part of Dashboard->apps page)

Reply:
I mean, the "All Apps" section is in the non-cloud Saleor App Store, the "install" button is just greyed out.

Reply:
After merge lets forward in to someone out of the team and check if these docs (in general, not only this point) is clear